src/components/BSelect/BSelect.vue

```diff
@@ -0,0 +1,154 @@
+<template>
+  <div>
+    <div>
+      <h4>{{ selectLabel }} <span v-if=isRequired aria-label='Required'>*</span></h4>
+    </div>
+    <div 
+      :aria-expanded="opened"
+      :aria-owns="id"
+      role="combobox"
+      tabindex="0"
+      v-model="selectLabel"
+    >
+      <ul
+        :aria-label="selectLabel"
+        :class="ulClass"
+        role="listbox"
+        :id="id"
+      >
+        <li 
+          :class="liClass" 
+          @click="openList"
+          role="option"
+          :style="selectedOption ? {color: '#000000'} : {color: '#cccccc'}"
```

Comment:
It is very frustrating when the comments go away. I personally don't like inline style, especially since the classes are already being computed so it would be easy (and is normally the case) to have a `.selected` style that gets added there.